databases/business-sync/fragment-shoebox-item.js

```diff
@@ -62,6 +62,51 @@
           }
         }
       }
+    },
+    annotations: {
+      type: 'array',
+      required: false,
+      arrayElementsValidator: {
+        mustNotBeEmpty: true,
+        type: 'object',
+        required: true,
+        propertyValidators: {
+          // indicates what type of annotation this is
+          // Only embedded is supported to start
+          type: {
+            type: 'string',
+            immutable: true,
+            predefinedValues: [ 'embedded' ],
+          },
+          // a simple tag describing what type data this annotation is
+          // example:  'user-metadata', 'ocr-analysis', 'model-XYZ-classification'
+          dataType: {
+            type: 'string',
+            immutable: true,
+            predefinedValues: [ 'user-metadata' ]
+          },
+          // payload, pretty much unrestricted at this point
+          data: {
+            type: 'object',
+            required: true
+          },
+          // timestamp recording when this annotation was last updated
+          lastModified: {
+            type: 'datetime',
+
+          },
+          // ID of the annotating user
+          annotatingUser: {
+            type: 'integer',
+            minimumValueExclusive: 0,
+          }
+        }
+      }
+    },
+    // indicating whether a record has been processed
+    processed: {
```

Comment:
I'm beginning to wonder if this field should also have a timestamp and whois info, since it is not inherently tied to any of the annotations and in theory could happen distinctly.  Essentially, do we want to capture more info about 'what was the final decision/action that moved this to processed' (which could be a combination of inputs from several annotations)?

Reply:
I think this probably is a good idea. On the books side we'd see a Changelog entry for the record. But if something ends up not being a record then we wouldn't have that. I'm +1